src/snowflake/snowpark/__init__.py

```diff
@@ -67,3 +67,35 @@
     WhenNotMatchedClause,
 )
 from snowflake.snowpark.window import Window, WindowSpec
+
+
+def register_sql_magic():
+    try:
```

Comment:
can you describe what is the purpose of this function or point me to some documentation I can follow

Reply:
the purpose of this function is that when an import is done it will run __init__.py and call register_sql_magic. This will function will try to gather the ipython environment and if found it will register the sql magic